### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MauriceHendrix**
## Description

Updated the readme and setup.py files in preparation for a trial release on test.pypy (beofre releasing propper)

## Motivation and Context

Tydying up hopefully with not too many errors

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

Documentation upddate

## Checklist:

- [X] I have updated all documentation necessary.
- [ ] I have checked spelling in (new) comments.

I've tried could could really do with somebody checking I've doen it properly & typos

## Testing
- [ ] Testing is done automatically and codecov shows test coverage
- [ ] This cannot be tested automatically 
n/a

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Contributing to cellmlmanip
 
+##Developer installation
+Users install chaste_codegen using pip. For instructions on installing as a user, see README.md
+
+For developers, to set up a virtual environment and install requirements:
+
+conda create -n cellmlmanip python=3.6
```
Comment (MichaelClerx) [inline CONTRIBUTING.md:8]: I don't think we should include virtual environment info here, especially not with conda (which is not a requirement for cellmlmanip)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Contributing to cellmlmanip
 
+##Developer installation
```
Comment (MichaelClerx) [inline CONTRIBUTING.md:3]: Space before Developer

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+
+For developers, to set up a virtual environment and install requirements:
+
+conda create -n cellmlmanip python=3.6
+conda activate cellmlmanip
+pip install -r requirements/dev.txt
+pip install -e .
+Cellmlmanip is tested and developed on Python 3.5 and higher.
+
+For more developer instructions, see CONTRIBUTING.md
```
Comment (MichaelClerx) [inline CONTRIBUTING.md:14]: That's this file

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+##Developer installation
+Users install chaste_codegen using pip. For instructions on installing as a user, see README.md
+
+For developers, to set up a virtual environment and install requirements:
+
+conda create -n cellmlmanip python=3.6
+conda activate cellmlmanip
+pip install -r requirements/dev.txt
+pip install -e .
+Cellmlmanip is tested and developed on Python 3.5 and higher.
```
Comment (MichaelClerx) [inline CONTRIBUTING.md:12]: This info (supported versions) belongs on the front page

File: README.md [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
 # cellmlmanip
-CellML loading and model equation manipulation
+CellML loading and model equation manipulation library
 
-## Developer installation
+## Installing 
+We reccomend installing chaste_codegen in a vritual environment (using conda or virtualenv)
 
-At the moment, there is no user installation procedure for cellmlmanip.
+Users install `cellmlmanip` using pip.
```
Comment (MichaelClerx) [inline README.md:9]: Probably just say "Cellmlmanip can be installed using pip:"

No need to differentiate between user/devel here. Also not sure it's called `cellmlmanip`, in code style, is it?

Comment (MauriceHendrix) [reply README.md:9]: do you have a better idea for naming the pypi pacakge?

Comment (MichaelClerx) [reply README.md:9]: OK, I understand what you're saying now :D

Comment (MichaelClerx) [reply README.md:9]: I would write something like "Users would install Cellmlmanip using `pip install cellmlmanip`"

File: setup.py [hunk 6]
```diff
@@ -12,8 +12,8 @@
     version='0.1.0',
     description='CellML loading and model equation manipulation',
     long_description=readme,
-    author='',
-    author_email='',
+    author=['Asif U Tamuri', 'Sarah M Keating', 'Maurice Hendrix', 'Michael Clerx', 'Jonathan Cooper'],
```
Comment (MichaelClerx) [inline setup.py:15]: Not sure we can do this https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py

File: README.md [hunk 7]
```diff
@@ -1,20 +1,21 @@
 [![Build Status](https://travis-ci.org/ModellingWebLab/cellmlmanip.svg?branch=develop)](https://travis-ci.org/ModellingWebLab/cellmlmanip) [![Documentation Status](https://readthedocs.org/projects/cellmlmanip/badge/?version=latest)](https://cellmlmanip.readthedocs.io/en/latest/?badge=latest) [![Coverage Status](https://codecov.io/gh/ModellingWebLab/cellmlmanip/branch/develop/graph/badge.svg)](https://codecov.io/gh/ModellingWebLab/cellmlmanip/)
 
 # cellmlmanip
-CellML loading and model equation manipulation
+CellML loading and model equation manipulation library
 
-## Developer installation
+## Installing 
+We reccomend installing chaste_codegen in a vritual environment (using conda or virtualenv)
```
Comment (jonc125) [inline README.md:7]:
```suggestion
We recommend installing cellmlmanip in a virtual environment (e.g. using conda or pipenv or virtualenv).
```

File: README.md [hunk 8]
```diff
@@ -1,20 +1,21 @@
...
-pip install -e .
+```
+pip install cellmlmanip
 ```
 
 Cellmlmanip is tested and developed on Python 3.5 and higher.
 
-For more developer instructions, see [CONTRIBUTING.md](./CONTRIBUTING.MD)
+## Documentation
+API documentation explaining how to use cellmlmanip can be found on [readthedocs](https://cellmlmanip.readthedocs.io/en/latest/?badge=latest)
```
Comment (jonc125) [inline README.md:18]:
```suggestion
API documentation explaining how to use cellmlmanip can be found on [readthedocs](https://cellmlmanip.readthedocs.io/en/latest)
```

**Pull request merged**